quarkus cli should give hint that you can run with `-e` to get error info

### Describe the bug

currently if you run `quarkus create app -x github-action` you get an error:

```
quarkus create app -x github-action xyzdkdk
-----------
selected extensions:
- io.quarkiverse.githubaction:quarkus-github-action

[ERROR] ❗  Unable to create project: Error while rendering template: /codestarts/quarkus/quarkiverse-github-action-codestart/base/action.docker.tpl.qute.yml
```

not a lot of help.

Took me a while to remember `-e` existed.

so it would be great if that error message would have  a hint like:

`Run with `-e` to get stacktraces and/or `--verbose` for more detailed logging`



### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/devtools/cli/src/main/java/io/quarkus/cli/common/OutputOptionMixin.java
+++ b/devtools/cli/src/main/java/io/quarkus/cli/common/OutputOptionMixin.java
@@ -162,6 +162,11 @@ public class OutputOptionMixin implements MessageWriter {
             CommandLine.UnmatchedArgumentException.printSuggestions((CommandLine.ParameterException) ex, out());
         }
         error(message);
+
+        if (!isShowErrors()) {
+            info("\nAdd the -e/--errors option to get more information about the error. Add the --verbose option to get even more details.");
+        }
+
         return cmd.getExitCodeExceptionMapper() != null ? cmd.getExitCodeExceptionMapper().getExitCode(ex)
                 : mixee.exitCodeOnInvalidInput();
     }

